easybuild/easyconfigs/b/binutils/binutils-2.25-GCC-4.9.2-binutils-2.25.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'binutils'
+version = '2.25'
+
+homepage = 'http://directory.fsf.org/project/binutils/'
+description = "binutils: GNU binary utilities"
+
+toolchain = {'name': 'GCC', 'version': '4.9.2-binutils-%(version)s'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+builddependencies = [
+    ('zlib', '1.2.8'),
+    ('binutils', version, '', True)
+]
+
+# make sure that system libraries are also considered by ld
+configopts = '--with-sysroot=/'
```

Comment:
please use same order as in other binutils easyconfig

also: should we somehow make it clear which binutils are statically linked against zlib (versionsuffix?), or link zlib statically in all binutils easyconfigs (at all the ones for version 2.25)